Formula/jsctl.rb

```diff
@@ -36,6 +36,6 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("jsctl version")
+    assert_match version.to_s, shell_output(bin/"jsctl version")
```

Comment:
is that right? 🤔 I'd have expected something like:

```suggestion
    assert_match version.to_s, shell_output("bin/jsctl version")
```